src/frontend/src/flows/login/unknownAnchor.ts

```diff
@@ -100,39 +109,41 @@ const initRecovery = (connection: Connection) => {
 
 const initLogin = (
   connection: Connection,
-  { userNumberInput }: { userNumberInput: HTMLInputElement },
+  { userNumberInput }: { userNumberInput: Ref<HTMLInputElement> },
   resolve: (res: LoginFlowResult) => void
 ) => {
   const loginButton = document.getElementById(
     "loginButton"
   ) as HTMLButtonElement;
 
-  userNumberInput.onkeypress = (e) => {
-    // submit if user hits enter
-    if (e.key === "Enter") {
-      e.preventDefault();
-      loginButton.click();
-    }
-  };
+  withRef(userNumberInput, (userNumberInput) => {
+    userNumberInput.onkeypress = (e) => {
```

Comment:
isn't this code duplicated? worth extracting a function?

Reply:
yes, will do in a separate PR

Reply:
you're right! updated